climate/ReadInGriddedData.R

```diff
@@ -0,0 +1,62 @@
+### READ IN GRIDDED CLIMATE DATA 2009 - XXX
+
+# Explanation for variables
+# TAM er temperatur (døgnmiddel)
+# UUM - rel.luftfuktighet
+# FFM - middelvind
+# NNM - midlere skydekke (i 8-deler)
+# RR - døgnnedbør
+
+# LIBRARIES
+library("dataDownloader")
+library("lubridate")
+library("tidyverse")
+library("ggplot2")
+
+
+# download data from OSF
```

Comment:
See how I have done this with here::here() in the database vignette, using objects for path and filename